_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,319 @@
+    continue to be used for fast and secure offchain payments while the
+    participants wait for the onchain splice-in or splice-out
+    transaction to confirm.
+
+    For more information, see the following threads: [1][zmn splicing
+    cut-through], [2][pickhardt bolt splicing], [3][russell splicing].
+    See also [Newsletter #17][].
+
+- **Wumbo:** by agreement among early LN implementations, currently each
+  channel's capacity is limited by default to about 0.168 BTC (about $40
+  USD when defined; currently about $1,000).  This was [chosen][russell
+  why limit] to help prevent users from putting too much money into
+  unproven software.
+
+    Several years later, LN has matured significantly and some
+    participants want to signal that they're willing to open higher
+    value channels.  The 1.1 spec proposal will allow such participants
+    to set a bit named "wumbo" (jumbo) to indicate their willingness to
+    accept larger channels and larger in-channel payments.
+
+    For more information, see the following threads: [1][zmn describing
+    wumbo], [2][zmn global wumbo].  For etymological reference, the name
+    wumbo appears to come from a [segment][youtube wumbo] in the
+    SpongeBob SquarePants cartoon where an "M" is interpreted as
+    standing for Mini, is inverted into a "W", and redefined as standing
+    for wumbo.
+
+- **Hidden destinations:** LN payments currently route payments using an
+  onion method similar to sending data to a Tor exit node.  Alice wants
+  to ultimately pay Zed, so she finds a path to him through Bob,
+  Charlene, and Dan.  To prevent the intermediaries from learning about
+  anything but the two channels they route though (e.g. Charlene knows
+  about Bob and Dan), Alice encrypts each step of the path so that only
+  the next step in disclosed to each recipient.  When Zed ultimately
+  receives the payment, he can simply return the success preimage to
+  Dan, who returns it to Charlene, and so forth back to Alice.
+
+    However, Tor also has a hidden service mode where the both the
+    sender and the recipient each choose part of the path so that
+    neither of them can determine exactly where the packets came from or
+    went---providing significantly improved privacy.  This proposal for
+    LN mirrors that mode.  Alice will still choose Bob, Charlene, and
+    Dan, but Zed can prevent Alice from learning about his routes by
+    choosing Edmond, Fran, and George.  Zed provides information about
+    how to find Edmond to Alice---but the information about Fran,
+    George, and Zed's own node is encrypted so that Alice can't see it.
+    This can allow hidden channels---a current feature of several LN
+    implementations---to stay hidden even when routing payments from
+    arbitrary spenders.
+
+    For more information, see the following threads where this feature
+    is also called rendez-vous routing: [1][cjp rz], [2][zmn rz], [3][zmn
+    rz packetswitch].
```

Comment:
For a detailed writeup you can point to https://github.com/lightningnetwork/lightning-rfc/wiki/Rendez-vous-mechanism-on-top-of-Sphinx. We're trying to aggregate the discussions in the Wiki so it should get easier to get a complete view of the proposal instead of having to read through the evolution in the form of mails :wink:

Reply:
Also if anybody has an idea on how to write that up in a more concise and precise way I'd be happy to hear :-)